### ISSUE TITLE
Improve documentation for 429 error

### ISSUE THREAD

**Issue by SpencerKaiser** [hjdhjd/homebridge-myq#1126]
# Preface
## **_Please read... I promise this isn't just another 429 auth issue duplicate!!!_**

---

**Describe The Problem:**
There's clearly a ton of users posting about the 429 issue and I think part of the issue is the error that's displayed in the plugin and in the logs...

### Here's a summary of where my head is at and why I think there's an issue with the verbiage:
- A 429 error usually indicates that a client is exceeding a rate limit
- Exceeding a rate limit and **_continuing to make requests_** typically _**extends**_ the lockout
- Users may see constant failures being logged out and think that the 60-90 min. lockout is being `extended` after each request/failure

**To Reproduce:**
Trigger the 429 lockout and observe the error in the Homebridge logs or on the plugin's `Feature Options` tab

**Logs:**
```
myQ API error: Temporary myQ API server-side issues encountered: 429 - Too Many Requests. This typically indicates a myQ API lockout for a 60-90 minute period before resuming API connectivity.
```

**Environment:**
* **Homebridge Version:** 1.6.1
* **Node Version:** 18
* **Homebridge-myQ Plugin Version**: 3.4.3
* **Operating System and OS Version:** Raspbian GNU/Linux Buster (10)

---

### Follow up Questions
- Will the lockout actually expire if the plugin is still spamming requests even though there's a 429?
- Does the user need to take any action or just wait? (I currently feel like maybe I need to just unplug my home bridge device for 90 mins...)
- Would it make sense to add a manual check to prevent constant failures after an initial 429?

**Comment by hjdhjd** [comment]
Thanks for the creative issue. 😄 

It does not extend the lockout. The user doesn't need to take action, other than to walk away for a couple of hours. If you're of the particularly paranoid variety, turn off the plugin for a few hours.

And I currently have no plans to intentionally backoff when a 429 error takes place - there's no meaningful way to tell how much time is left. So the "safe" thing to do would be to wait 60-90 minutes...whether it's truly 10 minutes remaining, or 50 minutes remaining. In a world with a bunch of bad options, I've made the choice to keep retrying until we succeed.

**Comment by SpencerKaiser** [comment]
> It does not extend the lockout.

Excellent... think it'd be worth adding this as part of the error message?

> I currently have no plans to intentionally backoff ...  I've made the choice to keep retrying until we succeed.

Got it, makes sense

> This is not about rate-limiting in the traditional sense.

Cool cool, so just _more_ misuse of HTTP status codes 😅 👌  

Thanks for the quick reply and thanks for the plugin! Hopefully things will get back to working here shortly... just updated today and made sure my auth credentials were correct 🤞

**Comment by SpencerKaiser** [comment]
FWIW I left it running for about 6 hours, it was still erroring, deactivated the plugin overnight, and now it works... thoughts?

**Comment by mattgahs** [comment]
Something else for users to consider (that @hjdhjd has no control over): If you're using another MyQ integration (Scrypted, Home Assistant, etc) in addition to Homebridge, that could be inducing 429s more frequently. 

I've been fighting with similar issues on the HA end of things, completely forgetting I also had MyQ in Homebridge for HomeKit. I may end up disabling the Homebridge version and pass the garage door to HomeKit via HA's HomeKit integration.

**Comment by LokiThorne** [comment]
I started getting this error last night.  Deactivated the plugin for the evening and restarted in the morning.  Same 429 errors

**Comment by nickmouaikel** [comment]
same issue as @LokiThorne. Even tried reverting to v3.4.2 and v3.4.1. Same 429 error too many requests. I even shutdown my server overnight and the problem persists. Also when I updated to v.3.4.3, my network activity skyrocketed and myq api was refreshing every 3 seconds or so. I noticed it said this in the documentation, but figured it would stop after 5 minutes or so, but continued past 5 minutes.
